src/main/java/io/github/railroad/Railroad.java

```diff
@@ -151,7 +157,7 @@ public void start(Stage primaryStage) {
         primaryStage.setMinWidth(scene.getWidth() + 10);
         primaryStage.setMinHeight(scene.getHeight() + 10);
         primaryStage.setScene(scene);
-        primaryStage.setTitle("Railroad - 1.0.0(dev)");
+        primaryStage.setTitle(localize("railroad.home.window.title") + " - 1.0.0(dev)");
```

Comment:
The title of the application should not be localized. 
```suggestion
        primaryStage.setTitle(localize("Railroad - 1.0.0(dev)");
```